app/presenters/notices/setup/check.presenter.js

```diff
@@ -31,17 +31,45 @@ function go(recipients, page, pagination, session) {
   const formattedRecipients = _recipients(noticeType, page, recipients, session.id)
 
   return {
+    backLink: _backLink(session),
     defaultPageSize,
     links: _links(session),
     pageTitle: _pageTitle(page, pagination),
+    pageTitleCaption: `Notice ${referenceCode}`,
     readyToSend: `${NOTIFICATION_TYPES[noticeType]} are ready to send.`,
     recipients: formattedRecipients,
     recipientsAmount: recipients.length,
-    referenceCode,
     warning: _warning(formattedRecipients)
   }
 }
 
+/**
+ * Check pages should not have backlinks.
+ *
+ * This page has them as some journeys do not have a prior check page.
+ *
+ * The ad hoc journey has a check notice type and so does not require a backlink.
+ *
+ * @private
+ */
+function _backLink(session) {
+  const { id, journey } = session
+
+  if (journey === 'adhoc') {
+    return null
+  } else if (journey === 'alerts') {
+    return {
+      href: `/system/notices/setup/${id}/abstraction-alerts/alert-email-address`,
```

Comment:
I think we it's better to just pass the link than the text and href and have the backlink just set the text.  

We call it a back link when passed in. 

If we have places where the text is different then lets just have that be specific to those pages.